common/on-chip-calib.cpp

```diff
@@ -33,8 +33,8 @@ namespace rs2
             if( val_in_range( device_name_string, { std::string( "Intel RealSense D415" ) } ) )
                 speed = 4;
 
-            if ( strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "ABCD") || strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "ABCE"))
-                is_mipi_sku = true;
+            if ( strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "ABCD") == 0 || strcmp(_dev.get_info(RS2_CAMERA_INFO_PRODUCT_ID), "ABCE") == 0)
```

Comment:
Please use device name, not PID

Reply:
Used connection type instead.
Will verify on both cameras

Reply:
Connection type is a great solution